server/test/article.spec.js

```diff
@@ -365,8 +368,48 @@ describe('Article email share', () => {
       .post(`/api/v1/article/${validSlug}/share`)
       .set({ Authorization: `Bearer ${userToken}` })
       .send(mockDataShare.validShareObject);
-
     expect(res).to.have.status(200);
     expect(res.body).to.have.property('message').eql('Article shared successfully');
   });
 });
+
+describe('Delete Article', () => {
+  before(async () => {
+    const userResponse = await chai
+      .request(app)
+      .post('/api/v1/auth/login')
+      .send({
+        email: 's.david@example.com',
+        password: 'soldier'
+      });
+
+    authorToken = userResponse.body.data.token;
+  });
```

Comment:
Looking through the codebase, there is already a before call made, why did you make multiple calls to the same login route? refactoring the first one to suit your need could be better

Reply:
Will work on this

Reply:
Done